dapr/proto/runtime/v1/dapr.proto

```diff
@@ -53,6 +53,9 @@ service Dapr {
   // Publishes events to the specific topic.
   rpc PublishEvent(PublishEventRequest) returns (google.protobuf.Empty) {}
 
+  // PublishActorEventAlpha1 Publishes an event from an actor with a specific topic.
+  rpc PublishActorEventAlpha1(PublishActorEventRequest) returns (google.protobuf.Empty) {}
```

Comment:
Please, move this close to the other Actor methods. On the client side, this belongs to the ActorProxy IMO.